src/components/LayoutTree.tsx

```diff
@@ -231,6 +233,11 @@ const LayoutTree = () => {
     },
   )
 
+  // compare between state.cursor and the position of the thought
+  const activeThought = useSelector(state =>
```

Comment:
Let's call this `cursorThoughtPositioned`. I'd like to avoid adding new terminology like "active."